en/docs/manage-apis/design/b2b-api-management/api-consumption.md

```diff
@@ -29,39 +29,50 @@ For more information on setting up WSO2 Identity Server 7.1.0, see [Setup WSO2 I
     keytool -import -alias wso2is7cert -file is7.cert -keystore client-truststore.jks -storepass wso2carbon
     ```
 
+2. Configure the `logoutSessionStateAppender` setting to `OIDC` in each portal if API Manager and WSO2 Identity Server are running on different hosts. Skip this step if both servers are running on localhost.
```

Comment:
Shall we change 'localhost' to 'the same host'? Also lets use same style when naming WSO2 products (ex: WSO2 API Manager and WSO2 Identity Server or API Manager and Identity Server).